domains/oreoluwa.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "oreoluwa-bs",
+        "email":"hello@oreoluwabs.com"
```

invalid email, if you don't want to share your email, you can safely delete it
```suggestion
        "username": "oreoluwa-bs"
```